src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php

```diff
@@ -38,6 +38,7 @@ public function __construct(
         private string $defaultLocale,
         private string $endpoint,
         private ?TranslatorBagInterface $translatorBag = null,
+        private string $restrictToStatus = 'translated,blank-translation',
```

Comment:
Should this be an array that we implode when necessary? Feels brittle to deal with strings, even more if argument validation is added at some point.

Reply:
Sure, would be more elegant. But idk if really necessary rn?

Reply:
Ah, also with `status` as a string you could use more complicated queries using ! for negation, etc.. The documentation says:
> Export translations with a specific status or flag. Negate values by prefixing with !. e.g. "translated", or "!fuzzy".

Reply:
> Sure, would be more elegant. But idk if really necessary rn?

Always nice to have things done the clean way? Also, it could be tricky to update it because of the BC promise once it's merged.

> also with status as a string you could use more complicated queries using ! for negation

Which is also possible with an array. If your implode delimiter is `,`, there's nothing to worry about to add `!` or any chatacter

Reply:
I would rather not touch it.
If I would, the DSN parsing would has to change as well. And I dont see this string vs array thing as a blocker for the problem I'm trying to solve. If the user would give a strange value as  `status`, the export API from loco would throw some (hopefully meaningful) exception that should be sufficient.